cypress/e2e/application-progress.cy.js

```diff
@@ -98,43 +94,144 @@ describe("View the application progress for a digital LPA", () => {
+        Cypress.$(elts[4]).find("svg[data-progress-indicator=not-started]")
+          .length,
       ).to.equal(1);
       expect(
-        Cypress.$(elts[5]).find("svg[data-progress-indicator=in-progress]")
+        Cypress.$(elts[5]).find("svg[data-progress-indicator=not-started]")
           .length,
       ).to.equal(1);
       expect(
         Cypress.$(elts[6]).find("svg[data-progress-indicator=not-started]")
           .length,
       ).to.equal(1);
+    });
+
+    cy.contains("Donor identity confirmation").click();
+    cy.contains("Not started");
+    cy.contains("Start donor identity check").should("not.exist");
+  });
+
+  it("shows application progress in progress", () => {
+    cy.addMock(`/lpa-api/v1/digital-lpas/${uid}/progress-indicators`, "GET", {
+      status: 200,
+      body: {
+        digitalLpaUid: uid,
+        progressIndicators: [
+          { indicator: "FEES", status: "IN_PROGRESS" },
+          { indicator: "DONOR_ID", status: "IN_PROGRESS" },
+          { indicator: "CERTIFICATE_PROVIDER_ID", status: "IN_PROGRESS" },
+          {
+            indicator: "CERTIFICATE_PROVIDER_SIGNATURE",
+            status: "IN_PROGRESS",
+          },
+          { indicator: "ATTORNEY_SIGNATURES", status: "IN_PROGRESS" },
+          { indicator: "PREREGISTRATION_NOTICES", status: "IN_PROGRESS" },
+          { indicator: "REGISTRATION_NOTICES", status: "IN_PROGRESS" },
+        ],
+      },
+    });
+
+    cy.visit(`/lpa/${uid}`);
+
+    cy.get(".app-progress-indicator-summary").then((elts) => {
+      expect(
+        Cypress.$(elts[0]).find("svg[data-progress-indicator=in-progress]")
+          .length,
+      ).to.equal(1);
+      expect(
+        Cypress.$(elts[1]).find("svg[data-progress-indicator=in-progress]")
+          .length,
+      ).to.equal(1);
       expect(
-        Cypress.$(elts[7]).find("svg[data-progress-indicator=not-started]")
+        Cypress.$(elts[2]).find("svg[data-progress-indicator=in-progress]")
           .length,
       ).to.equal(1);
       expect(
-        Cypress.$(elts[8]).find("svg[data-progress-indicator=not-started]")
+        Cypress.$(elts[3]).find("svg[data-progress-indicator=in-progress]")
           .length,
       ).to.equal(1);
       expect(
-        Cypress.$(elts[9]).find("svg[data-progress-indicator=not-started]")
+        Cypress.$(elts[4]).find("svg[data-progress-indicator=in-progress]")
           .length,
       ).to.equal(1);
       expect(
-        Cypress.$(elts[10]).find("svg[data-progress-indicator=not-started]")
+        Cypress.$(elts[5]).find("svg[data-progress-indicator=in-progress]")
+          .length,
+      ).to.equal(1);
+      expect(
+        Cypress.$(elts[6]).find("svg[data-progress-indicator=in-progress]")
           .length,
       ).to.equal(1);
     });
+
+    cy.contains("Donor identity confirmation").click();
+    cy.contains("In progress");
+    cy.contains("Start donor identity check").should("exist");
```

Comment:
Is it worth a check on the content of the link href? Or do you think that's too much?